.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--
+Thanks for contributing!
+-->
+
+Some things to make sure you've done:
```

Comment:
It might make sense to put all of this into the comment, since it's not text we'd want to show up in the visible PR text. If that's not the case, you could consider turning these into [task lists](https://help.github.com/articles/about-task-lists/#creating-task-lists) with checkbox syntax, which would allow users to check them off manually.

An alternative idea is to ask users to report things such as BC breaks and confirm that tests pass. One example of this is the [Symfony PHP framework's template](https://github.com/symfony/symfony/blob/master/.github/PULL_REQUEST_TEMPLATE.md).

Reply:
Wow didn't even know you could use these "task lists". Incredible, I've done that.